crates/ibc/src/hosts/tendermint_helper.rs

```diff
@@ -15,62 +15,73 @@ use tendermint::trust_threshold::TrustThresholdFraction as TendermintTrustThresh
 
 /// Provides an implementation of `ValidationContext::validate_self_client` for
 /// Tendermint-based hosts.
-pub trait ValidateSelfClientContext {
-    fn validate_self_client(&self, counterparty_client_state: Any) -> Result<(), ConnectionError> {
-        let counterparty_client_state = TmClientState::try_from(counterparty_client_state)
-            .map_err(|_| ConnectionError::InvalidClientState {
-                reason: "client must be a tendermint client".to_string(),
+pub trait ValidateSelfTendermintClientContext {
```

Comment:
Since we are in the `tendermint_helper` module, I think we should remove `Tendermint` from the name.

Reply:
[907f6b2](https://github.com/cosmos/ibc-rs/pull/436/commits/907f6b24a815a8f2fa33f4aa702b67d791f25105)